plugins/circleci/src/pages/BuildsPage/lib/Builds/Builds.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Spotify AB
```

Comment:
Remove double header.

Reply:
I'm not sure what happens here but the linter complains and the second header was inserted by autofix.
```
$ backstage-cli lint
error: Missing notice header (notice/notice) at src/pages/BuildsPage/lib/Builds/Builds.tsx:17:1:
  15 |  */
  16 | 
> 17 | import React, { FC, useEffect } from 'react';
     | ^
  18 | 
  19 | import {} from 'circleci-api';
  20 | import { useApi } from '@backstage/core';


1 error found.
1 error potentially fixable with the `--fix` option.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Got it. There was a typo.

Reply:
The second header doesn't match the template. You should be able to remove that one.